docs/source/install-script.rst

```diff
@@ -0,0 +1,141 @@
+:orphan: true
+
+.. role:: bash(code)
+   :language: bash
+
+=================
+Firedrake Install
+=================
+
+The structure of the Firedrake install process consists of the following steps:
+
+1. Installing system dependencies
+2. Building and configuring PETSc
+3. Installing Python dependencies  
+
+These steps are only supported for for Linux and MacOS distributions. 
+Further we will take the above steps and discuss them in detail. 
+
+Installing system dependencies
+-------------------------------
+
+Linux
+~~~~~
+
+For Linux distributions the system dependencies are installed via the :bash:`apt-get` package manager. The necessary dependencies for running default firedrake are:
+
+- build-essential
+- autoconf
+- automake
+- bison
+- flex
+- cmake
+- gfortran
+- git
+- libtool
+- python3-dev
+- python3-pip
+- python3-tk
+- python3-venv
+- zlib1g-dev
+- libboost-dev
+- libopenblas-dev (if no configuration is provided by the user)
```

Comment:
Should these also use backticks like the MacOS packages below?

Reply:
Or I can just remove the backticks from the MacOS packages :). forgot to format